oic.r.sacl.raml

```diff
@@ -56,44 +54,31 @@ traits:
                           "recurrence": [ "RRULE:FREQ=DAILY;UNTIL=20180131T140000Z;BYMONTH=1" ]
                         }
                       ]
-                    }
-                  ],
-                  "aces2": [
+                    },
                     {
+                      "aceid": 2,
                       "subject": {
-                        "rl": {
                           "authority": "484b8a51-cb23-46c0-a5f1-b4aebef50ebe",
-                          "roles": [
-                            "SOME_STRING", "SOME_STRING"
-                          ]
-                        }
-                      },
+                          "role": "SOME_STRING"
+                        },
                       "resources": [
                         {
-                          "href": "coaps://IP-ADDR/light",
-                          "rel": "some-rel",
+                          "href": "/light",
                           "rt": ["oic.r.light"],
                           "if": ["oic.if.a"]
                         },
                         {
-                          "href": "coaps://IP-ADDR/door",
-                          "rel": "some-rel",
+                          "href": "/door",
                           "rt": ["oic.r.door"],
                           "if": ["oic.if.a"]
                         }
                       ],
                       "permission": 15
                     }
-                  ]
-
-                },
+               ],
                 "signature": {
                   "sigtype": "oic.sec.sigtype.pk7",
                   "sigvalue": "ENCODED-SIGNATURE-VALUE"
-                },
-                "amsuuid": "484b8a51-cb23-46c0-a5f1-b4aebef50ebe",
-                "ams": {
-                  "sl": [ "oic.sec.svc.doxs", "oic.sec.svc.bss" ]
                 }
```

Comment:
Should not it be oic.r.sacl.aclist2.subject to define the owner of the resource?